js/node/CMakeLists.txt

```diff
@@ -76,6 +100,9 @@ if (WIN32)
     COMMAND ${CMAKE_COMMAND} -E copy
       ${ONNXRUNTIME_BUILD_DIR}/${CMAKE_BUILD_TYPE}/onnxruntime.dll
       ${dist_folder}
+    COMMAND ${CMAKE_COMMAND} -E copy
```

Comment:
Do it only when onnxruntime_USE_DML is ON. Otherwise ${ONNXRUNTIME_BUILD_DIR}/${CMAKE_BUILD_TYPE}/DirectML.dll doesn't exist

Reply:
Done